kubernetes/provider.go

```diff
@@ -280,10 +384,6 @@ func initializeConfiguration(d *schema.ResourceData) (*restclient.Config, error)
 		for _, p := range v {
 			configPaths = append(configPaths, p.(string))
 		}
-	} else if v := os.Getenv("KUBE_CONFIG_PATHS"); v != "" {
-		// NOTE we have to do this here because the schema
-		// does not yet allow you to set a default for a TypeList
-		configPaths = filepath.SplitList(v)
```

Comment:
The SDK now seems to allow us to set a default for TypeList! :tada: I didn't see the feature added, but it worked for me when I defined my own function of type `schema.EnvDefaultFunc`.